src/gui/systray.cpp

```diff
@@ -374,26 +380,23 @@ QPoint Systray::computeWindowPosition(int width, int height) const
     const auto windowRect = [=]() {
         const auto rect = QRect(topLeft, bottomRight);
         auto offset = QPoint();
-
```

Comment:
nitpick: please don't remove those empty lines, this has nothing to do with the rest of your patch and reduced readability IMHO.